frame/staking/src/pallet/impls.rs

```diff
@@ -1276,19 +1294,23 @@ impl<T: Config> VoteWeightProvider<T::AccountId> for Pallet<T> {
 /// A simple voter list implementation that does not require any additional pallets. Note, this
 /// does not provided nominators in sorted ordered. If you desire nominators in a sorted order take
 /// a look at [`pallet-bags-list].
-pub struct UseNominatorsMap<T>(sp_std::marker::PhantomData<T>);
```

Comment:
Would be nice if you did a search in replace for `UseNominatorsMap` with `UseNominatorsAndValidatorsMap`. I think there might be a few comments in the codebase that will need updating